lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1027,7 +1027,7 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
+    Generate = [{rsa, 2048, 3}, {rsa, 3072, 65537}],
```

Comment:
Hm. 2k should be okay, from everything I've read, so I'd rather not remove the test without a better understanding of why it's failing. What error message does it throw?

I'll see if I can build a FIPS build and investigate locally, but I probably won't have time for a few days / a week.